walkthroughs/howto-k8s-timeout-policy/README.md

```diff
@@ -0,0 +1,131 @@
+## Overview
+This example shows how to configure App Mesh Route and VirtualNode listener timeouts. This feature allows us to specify a custom timeout value based on the application's need and when not specified a default timeout of 15 seconds will be applied to all the requests.
+
+We use AWS Cloud Map based service discovery mechanism in this example walkthrough.
+
+### Color
+There are two deployments of colorapp, _blue_ and _red_. Pods of both these deployments are registered behind service colorapp.howto-k8s-timeout-policy.pvt.aws.local. Blue pods are registered with the mesh as colorapp-blue virtual-node and red pods as colorapp-red virtual-node. These virtual-nodes are configured to use AWS CloudMap as service-discovery, hence the IP addresses of these pods are registered with the CloudMap service with corresponding attributes. We specify a timeout value of 60 secs for all the VirtualNodes and VirtualRouters.
+
+Additionally a colorapp virtual-service is defined that routes traffic to blue and red virtual-nodes.
+
+### Front
+Front app acts as a gateway that makes remote calls to colorapp. Front app has single deployment with pods registered with the mesh as _front_ virtual-node. This virtual-node uses colorapp virtual-service as backend. This configures Envoy injected into front pod to use App Mesh's EDS to discover colorapp endpoints.
+
+## Prerequisites
+[Walkthrough: App Mesh with EKS](../eks/)
+
+Note: v1beta2 example manifest requires [aws-app-mesh-controller-for-k8s](https://github.com/aws/aws-app-mesh-controller-for-k8s) version [>=v1.0.0](https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/CHANGELOG.md). Run the following to check the version of controller you are running.
+```
+$ kubectl get deployment -n appmesh-system appmesh-controller-manager -o json  | jq -r ".spec.template.spec.containers[].image" | cut -f2 -d ':'
```

Comment:
`appmesh-controller-manager` -> `appmesh-manager` as per the helm charts [1]

[1] https://github.com/aws/eks-charts/tree/master/stable/appmesh-manager